features/cli/policy.feature

```diff
@@ -60,10 +60,13 @@ Feature: change the policy of user/service account
 
     Given I have a project
     And I switch to the second user
+    And I wait up to 100 seconds for the steps to pass:
+    """
     When I run the :get client command with:
       | resource | project |
     Then the output should match:
```

Comment:
we can replace 65-68 with `And the expression should be true> project(project.name).active?`

Reply:
Good point, thanks! New logs: https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/92738/console

Reply:
@barleyer thanks!  
/lgtm